crates/biome_analyze/src/suppressions.rs

```diff
@@ -182,24 +178,22 @@ pub(crate) struct RangeSuppressions {
 pub(crate) struct RangeSuppression {
     /// Whether the current suppression should suppress all signals
     pub(crate) suppressed_categories: RuleCategories,
-
     /// The range of the `biome-ignore-start` suppressions
     pub(crate) start_comment_range: TextRange,
-
     /// A range that indicates how long this suppression has effect
     pub(crate) suppression_range: TextRange,
-
     /// Set to `true` when this line suppresses a signal that was already suppressed by another entity e.g. top-level suppression
     pub(crate) already_suppressed: Option<TextRange>,
-
     /// Whether this suppression has suppressed a signal
     pub(crate) did_suppress_signal: bool,
-
     /// Indicates if this suppression has found its end comment - if false, the suppression_range is not yet complete
     pub(crate) is_ended: bool,
-
```

Comment:
:(

Reply:
@arendjr You mean you would keep these brank lines? I removed them because of the inconsistency with `LineSuppression` and `RangeSuppresion` if you are curious.